### PR TITLE
Refunds Resend notification api

### PR DESCRIPTION
**Pull request by Jennings-HMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/functionalTest/resources/application-functional.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ auth:
 test:
   url :
     refunds : ${TEST_URL:http://localhost:8080}
-    payments : ${TEST_URL_PAYMENT:http://payment-api-pr-972.service.core-compute-preview.internal}
+    payments : ${TEST_URL_PAYMENT:http://payment-api-pr-1089.service.core-compute-preview.internal}
```
Comment (YasharRahvar) [inline src/functionalTest/resources/application-functional.yaml:53]: this needs to get updated to aat before merging to master

File: src/main/resources/application-local.yaml [hunk 2]
```diff
@@ -105,3 +105,23 @@ refunds:
     scope: ${REFUND_SERVICE_ACCOUNT_SCOPE:}
     redirectUri: ${REFUND_SERVICE_ACCOUNT_REDIRECT_URI:}
 
+notification:
+  url: ${NOTIFICATION_API_URL:http://localhost:8185}
+  letter-path: ${NOTIFICATION_LETTER_PATH:/letterNotification}
+  email-path: ${NOTIFICATION_EMAIL_PATH:/emailNotification}
+  email-to-reply: ${EMAIL_TO_REPLY:j@dummy.com}
```
Comment (YasharRahvar) [inline src/main/resources/application-local.yaml:112]: remove j@dummy.com

Comment (YasharRahvar) [inline src/main/resources/application-local.yaml:112]: remove dummy

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -176,3 +176,16 @@ logging:
         web: DEBUG
 
 
+notification:
+  url: ${NOTIFICATION_API_URL:http://localhost:4300}
+  letter-path: ${NOTIFICATION_LETTER_PATH:/letter}
+  email-path: ${NOTIFICATION_EMAIL_PATH:/email}
+  email-to-reply: ${EMAIL_TO_REPLY:j@dummy.com}
```
Comment (YasharRahvar) [inline src/main/resources/application.yaml:183]: remove j@dummy.com

Comment (YasharRahvar) [reply src/main/resources/application.yaml:183]: remove dummy

File: charts/ccpay-refunds-api/values.preview.template.yaml [hunk 4]
```diff
@@ -13,7 +13,7 @@ java:
     SPRING_LIQUIBASE_ENABLED: true
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat2.internal:8443/openam/oauth2/hmcts
     LAUNCH_DARKLY_USER_NAME_PREFIX: aat
-    PAYMENT_API_URL: "http://payment-api-pr-972.service.core-compute-preview.internal"
+    PAYMENT_API_URL: "http://payment-api-pr-1089.service.core-compute-preview.internal"
```
[the remark this answers is not in the document]
Comment (sayali-salunkhe) [reply charts/ccpay-refunds-api/values.preview.template.yaml:16]: True, will change it at the time of AAT deployment

File: src/main/resources/application-local.yaml (hunk 2 above)
Comment (sayali-salunkhe) [reply src/main/resources/application-local.yaml:112]: Done

Comment (sayali-salunkhe) [reply src/main/resources/application-local.yaml:112]: Done

File: src/main/resources/application.yaml (hunk 3 above)
Comment (sayali-salunkhe) [reply src/main/resources/application.yaml:183]: Done

File: src/functionalTest/resources/application-functional.yaml (hunk 1 above)
Comment (sayali-salunkhe) [reply src/functionalTest/resources/application-functional.yaml:53]: True, will change it at the time of AAT deployment

File: src/main/java/uk/gov/hmcts/reform/refunds/config/RestTemplateConfiguration.java [hunk 5]
```diff
@@ -27,4 +27,9 @@ public RestTemplate restTemplatePayment() {
         return new RestTemplate(new HttpComponentsClientHttpRequestFactory());
     }
 
+    @Bean("restTemplateNotify")
+    public RestTemplate restTemplateNotify() {
```
[the remark this answers is not in the document]
Comment (sayali-salunkhe) [reply src/main/java/uk/gov/hmcts/reform/refunds/config/RestTemplateConfiguration.java:31]: Done

**Comment by AnilDhawaleGitHub** [comment]
closing this pr

**Pull request closed without merging**